src/Microsoft.Diagnostics.Monitoring.StartupHook/StartupHook.cs

```diff
@@ -24,6 +25,14 @@ public static void Initialize()
             }
 
             s_exceptionProcessor.Start();
+
+            try
+            {
+                SharedInternals.MessageDispatcher = new MonitorMessageDispatcher(new ProfilerMessageSource());
+            }
+            catch
+            {
```

Comment:
Do we have any way of capturing errors from the Startup hook? I know that's not specific to your feature even, but I am concerned we have a lot of possible failure conditions and no way of knowing they are happening.